src/websockets/binary_websockets.es6

```diff
@@ -11,29 +11,24 @@ let socket = null;
 let is_website_up = false;
 let queued_requests = {};
 
-const get_app_id = () => {
-   const app_ids = JSON.parse(app_ids_json);
-   const config = local_storage.get('config');
-   let token = (config && config.app_id) || '';
-
-   if(!token) { /* find the appropriate token */
-      const href = window.location.href;
-      for(const web_address in app_ids) {
-         if(href.lastIndexOf(web_address, 0) == 0) {
-            token = app_ids[web_address];
-            break;
-         }
-      }
+const get_app_id = () => window.localStorage.getItem('config.app_id') || (/staging\.binary\.com/i.test(window.location.hostname) ? '1098' : '1');
```

Comment:
@nazaninreihani I have two questions

1. get_app_id() earlier and get_app_id() that you coded, does not look the same. Looks like they are doing something different.
2. config/config in webtrader was meant to be used by QA to change the endpoint. Now with your new work of allowing QA to change endpoint using chrome ext, I dont think we need config/config module of webtrader.
3. I see you are using 1098 in both next-gen and webtrader. Will it work? App Ids for these apps are different

Reply:
@arnabk You were right, I changed the extension code, so now first I check if the default app_id exists in 'oauth/app_id.json' and if true I set that in storage so the extension can read the default app_id, if the url is not any of the urls mentioned in 'oauth/app_id.json' the default app_id would be the default app_id for webtrader.
just to mention that this extension works only on these urls (containing staging.binary.com, app.binary.com, ticktrade.binary.com, developers.binary.com, www.binary.com and webtrader.binary.com).
which means in localhost the extension is off, (app_id can be changed by changing it in 'oauth/app_id.json'), and the default app_id will be defaults in 'oauth/app_id.json' || webtrader's default. (#1 and #3)

about #2, do you mean I can remove the config file?